src/util/Constants.js

```diff
@@ -7,10 +7,10 @@ exports.UserAgent = `DiscordBot (${Package.homepage.split('#')[0]}, ${Package.ve
 
 exports.WSCodes = {
```

Comment:
@ImRodry The `WSCodes` names below needs to be reverted.

Reply:
oops I did a mass replace and didn't notice those, will fix!